packages/backend/src/core/liveness/LivenessIndexer.ts

```diff
@@ -40,30 +45,35 @@ export class LivenessIndexer extends ChildIndexer {
   }
 
   override async update(from: number, to: number): Promise<number> {
-    const configurations = await this.livenessConfigurationRepository.getAll()
-
-    const { data, adjustedTo, usedConfigurationsIds } =
-      await this.livenessClient.getLivenessData(
-        this.projects,
-        configurations,
-        new UnixTime(from),
-        new UnixTime(to),
-      )
+    const { transfersConfig, functionCallsConfig, adjustedTo } =
+      await this.getConfiguration(from, to)
+
+    const data = await this.livenessClient.getLivenessData(
```

Comment:
if both configurations are empty do not call liveness client, instead just log that "Update skipped" {from, adjustedTo}